crawler/mesos.py

```diff
@@ -0,0 +1,237 @@
+#! /usr/bin/python
```

Comment:
Code coverage for this file is too low. There are a bunch of functions that dont seem to be used by the test nor the snaoshot() function. You could remove those.